import/src/test/java/com/google/cloud/healthcare/imaging/dicomadapter/AttributesUtilTest.java

```diff
@@ -66,7 +66,7 @@ public void testAttributesToQidoPath_urlEncode() throws Exception {
     String result = AttributesUtil.attributesToQidoPath(attrs);
 
     assertThat(result).isEqualTo(
-        "instances?limit=50000&" + TagUtils.toHexString(Tag.PatientName) + "=%25%26%23%5E+&");
+        "instances?limit=50000&" + TagUtils.toHexString(Tag.PatientName) + "=%25%26%23%5E&");
```

Comment:
Not sure what this change means?

Reply:
This changed with dcm4che version update. Seems to do url encoding slightly differently.

Reply:
Is there any behavior change?

Reply:
If not, this change is good with me

Reply:
After checking it with few strings: difference is that now trailing(!) spaces are dropped when setting dcm4che string attribute value. Should be safe.